src/main/java/medicaldoctor/Program.java

```diff
@@ -1,18 +1,27 @@
 package medicaldoctor;
 
+import medicaldoctor.controllers.LookUp;
+import medicaldoctor.controllers.ControllerManager;
+import com.sun.glass.ui.Screen;
 import javafx.application.Application;
-import javafx.fxml.FXMLLoader;
-import javafx.scene.Parent;
+import javafx.scene.Group;
 import javafx.scene.Scene;
 import javafx.stage.Stage;
 import medicaldoctor.core.DatabaseScope;
 
 public class Program extends Application {
-
     @Override
     public void start(Stage stage) throws Exception {
-        Parent root = FXMLLoader.load(getClass().getClassLoader().getResource("Login.fxml"));
+        ControllerManager cm = new ControllerManager();
```

Comment:
That is the pattern I have been using instead of some version of dependency injection, what you have in the `setScreenParent(ControllerManager page)`'s

Reply:
Then we don't need to worry about passing the cm all around.

Reply:
What do you mean by that?

Reply:
your ParentController interface

Reply:
@Sal-Hussain